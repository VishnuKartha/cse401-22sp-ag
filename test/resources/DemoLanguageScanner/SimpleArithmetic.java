a = b + c;
display a;