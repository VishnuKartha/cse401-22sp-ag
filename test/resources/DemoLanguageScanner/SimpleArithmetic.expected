ID(a) BECOMES ID(b) PLUS ID(c) SEMICOLON DISPLAY ID(a) SEMICOLON